diagnostic.md

```diff
@@ -45,6 +57,8 @@ weather === "sunny"
 
 What do these expressions evaluate to?  Explain your answers.
 
+true b/c === is an "evaluation" to test if the variable weather
+is set to 'sunny'
```

Comment:
Yes, also `let weather` evaluates to undefined and `weather = 'sunny'` evaluates to 'sunny'